Library/Homebrew/unpack_strategy.rb

```diff
@@ -48,6 +48,7 @@ def self.strategies
       Pax,
       Gzip,
       Lzma,
+      Dmg, # Needs to be before `Bzip2/Xz`.
```

Comment:
Since `man hdiutil` mentions LZMA, I think this should also be before `Lzma`.

Reply:
Yup, totes agree!

Reply:
Will re-test, following this tweak

Reply:
Still installs fine, following moving to above `Lzma`